rook/migration/v1.14/README.md

```diff
@@ -0,0 +1,52 @@
+# Upgrade rook-ceph to v1.14.x
+
+## Prerequisites
+
+- [ ] Check the state of rook-ceph so it is healthy before you begin:
+
+  ```bash
+  helm -n rook-ceph list --all
+  # should show all deployed
+  kubectl -n rook-ceph get jobs
+  # should show all completed
+  kubectl -n rook-ceph get pods
+  # should show all running or completed
+  kubectl -n rook-ceph get cephclusters
+  # should show PHASE = Ready and HEALTH = HEALTH_OK
+  ```
+
+## Apply upgrade - _disruptive_
+
+- 1. Fetch chart version
+
+```console
+helm fetch rook-release/rook-ceph --version v1.14.3 --untar --destination helmfile.d/upstream/rook-ceph
+helm fetch rook-release/rook-ceph-cluster --version v1.14.3 --untar --destination helmfile.d/upstream/rook-ceph-cluster
```

Comment:
This should be equivalent to doing a `git restore helmdile.d/upstream` right?

Reply:
Hmmm, per definition yes :thinking: